pheps/phep-0001.md

```diff
@@ -0,0 +1,287 @@
+
+"Consensus" here means no concrete objections have been raised: an accepted PHEP must be one that the entire community can at least live with. The PHEP authors must make a compelling case for the PHEP; objectors must provide meaningful concerns that could, in principle, be addressed by the authors. All are expected to discuss in good faith, while recognizing that good faith does not guarantee consensus can be reached.
+
+For a PHEP to be accepted it must meet certain minimum criteria. It must be a clear and complete description of the proposed enhancement. The enhancement must represent a net improvement. The proposed implementation, if applicable, must be solid and must not complicate PyHC projects unduly. Finally, a proposed enhancement must be "pythonic" in order to be accepted by the community. (However, "pythonic" is an imprecise term; it may be defined as whatever is acceptable to the community. This logic is intentionally circular.)
+
+The editors must post oronouncements of PHEP resolution to the PyHC mailing list.
+
+Once a PHEP has been accepted, the reference implementation, if applicable, must be completed. When the reference implementation is complete and incorporated into a PyHC project's source code repository, the status will be changed to "Final". PHEPs which do not require a reference implementation move immediately to "Final" upon acceptance.
+
+If a PHEP requires additional design and interface feedback before committing to long term stability, it must remain in "Draft". Alternatively, its scope can be reduced (e.g. by deferring some features to later PHEPs).
+
+If no progress is being made on the PHEP, the editor should consult the author. The author may resume work on the PHEP, [transfer it to another author](transferring-phep-ownership), or withdraw it. If the author takes none of these actions, the editor may choose to bring the PHEP to the community for approval or rejection in its current form.
+
+If a PHEP fails to find consensus, the author may choose to modify it for reconsideration in the future. The PHEP may also be "Withdrawn" if the PHEP author themself has decided that the PHEP is actually a bad idea, or has accepted that a competing proposal is a better alternative. A PHEP can also be "Rejected" if it fails to find consensus and the author is not willing or able to modify it. Perhaps after all is said and done it was not a good idea. It is still important to have a record of this fact.
+
+When a PHEP changes state, the PHEP text must be updated accordingly. In addition to updating the Status field, at the very least the Resolution header should be added with a direct link to the minutes of the telecons and meetings where a decision on the PHEP was made.
+
+Final PHEPs can also be replaced by a different PHEP, rendering the original obsolete.
+
+The possible paths of the status of PHEPs are as follows:
```

Comment:
I reduced the number of states relative to PEP 1 for simplicity: deferred and provisional are removed. I also renamed "superseded" to "replaced"; the two were used interchangeably in PEP 1